src/boot/bootctl.c

```diff
@@ -1416,6 +1454,17 @@ static int parse_argv(int argc, char *argv[]) {
                         }
                         break;
 
+                case 'a':
+                        if (streq(optarg, "auto"))
+                                arg_arch = mfree(arg_arch);
+                        else if (STR_IN_SET(optarg, "all", "ia32", "x64", "arm", "aa64", "riscv64")) {
+                                r = free_and_strdup(&arg_arch, optarg);
+                                if (r < 0)
+                                        return log_oom();
+                        } else
+                                return log_error_errno(SYNTHETIC_ERRNO(EINVAL), "Unknown EFI arch: %s", optarg);
+                        break;
```

Comment:
i have the suspicion this should actually permit being used multiple times, i.e. be an strv, not a simple string.

And I still think that if unspecified we should copy boot loaders for all archs for which one is installed. i.e. if people don't want some arch they shouldn't install the package for it, or conversely: installing the package for an arch should make the boot lodaer just work for that arch.